app/src/main/java/com/ottfstudio/quizdroid/presentation/home/HomeViewModel.kt

```
@@ -34,4 +59,42 @@ class HomeViewModel
+            fetchCorrectRate(count)
+        }
+
+        private fun fetchToday(): String {
+            return try {
+                val currentDate = LocalDate.now()
+                val formatter = DateTimeFormatter.ofPattern("yyyy.MM.dd")
+                val formattedDate = currentDate.format(formatter)
+                require(formattedDate.isNotBlank()) { "Formatted date is blank" }
+                _state.value = _state.value.copy(today = formattedDate)
+                formattedDate
+            } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)